pages/data-providers/[data-provider-id]/overview.jsx

```diff
@@ -87,6 +87,9 @@ const DataStatisticsCard = ({
 
 const DepositingCard = ({ chartData, complianceLevel }) => {
   const { title, description, action } = depositing
+
+  if (chartData.length === 0) return null
```

Comment:
Should not be here. It's not a problem of card. It's more about the page.

Reply:
What do you propose? Do you rather want to show a card without chart? If so it will show wrong data in the description. Should I replace `100%` -> `unknown.`

```
You have about 100% non-compliant works.
```

Reply:
You can do it one level above, in `DashboardView` component. 

Either create placeholder card that would display _We are collecting your data. It will appear soon here_ and display it if data absent or add such test into the card and show text inside. 

Despite repeating some parts I would go first way.

Reply:
https://github.com/oacore/dashboard/pull/95/commits/c9b79e5237a71db01f58645a6a9f349a74b0bf38